kansha/board/models.py

```diff
@@ -180,6 +181,23 @@ def get_pending_users(self):
     def set_background_image(self, image):
         self.background_image = image or u''
 
+    def delete_column(self, column):
+        """Delete column
+
+        Delete a given column, re-index other columns and delete all cards left
+        in the column
+
+        In:
+            - ``column`` -- DataColumn instance to delete
+        """
+        index = column.index
+        column.delete()
+        session.flush()
+        q = DataColumn.query
+        q = q.filter(DataColumn.index >= index)
+        q = q.filter(DataColumn.board == self)
+        q.update({'index': DataColumn.index - 1})
+
     @classmethod
     def get_last_modified_boards_for(cls, user_username, user_source):
```

Comment:
Must be method `get_last_modified_boards()` of `User`

We can't access the data model of the `History` component in the model of `Board`

Why manually build SQL queries instead of using the ORM?